doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,241 @@
+
+Logical components
+----------------------------------------------------
+
+Plutus tools consists of libraries, executables and logical components within Haskell packages for external developers to use. 
+A single Haskell package may contain multiple logical components. 
+
+The tools are located in the `plutus-apps <https://github.com/input-output-hk/plutus-apps>`_ repository. 
+
+For each tool or logical component shown below, we have indicated its specific location within plutus-apps and provided a brief description. 
+
+.. figure:: ./plutus-tools-ecosystem.png
+
+    Illustration of the Plutus Tools Ecosystem showing logical components and some indication of their dependencies and relationships. 
+    
+    *QUESTION: Would it be appropriate to map the order in which we list the Tools on this page with the approximate order in which they are shown in the diagram?*
+
+1. Marconi
+~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+---------------------------------------------------------------+
+| Marconi                                                                      |
++==============+===============================================================+
+| Location     | ``plutus-apps/marconi``                                       |
++--------------+---------------------------------------------------------------+
+
+`Marconi <https://github.com/input-output-hk/plutus-apps/tree/main/marconi>`_ is an application for indexing data from the Cardano blockchain. 
+`Marconi` is faster and more flexible than `chain-index` and will eventually replace it. 
+`Marconi` is currently in an early alpha version. It has several indexers. 
+While much of the architecture is settled, some aspects are in the design stage. 
+
+2. Plutus contract model testing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
++--------------+---------------------------------------------------------------+
+| Plutus contract model testing                                                |
++==============+===============================================================+
+| Location     | ``plutus-apps/plutus-contract``                               |
++--------------+---------------------------------------------------------------+
+
+`Plutus contract testing model <https://github.com/input-output-hk/plutus-apps/blob/main/plutus-contract/src/Plutus/Contract/Test/ContractModel.hs>`_ is used for testing prototype Plutus contracts with *contract models*, using the framework provided by :hsmod:`Plutus.Contract.Test.ContractModel`. 
+This framework generates and runs tests on the Plutus emulator, where each test may involve a number of emulated wallets, each running a collection of Plutus contracts, all submitting transactions to an emulated blockchain. 
+Once you have defined a suitable model, then QuickCheck can generate and run many thousands of scenarios, taking the application through a wide variety of states, and checking that it behaves correctly in each one. 
+
+   See also: 
```

Comment:
Not sure but:
See also: -> See the following tutorial:

Reply:
Good idea.